.circleci/config.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - run:
           name: run tests
           command: |
-            tox
+            tox -e test test_no_extras
```

Comment:
Can you confirm that the `test_no_extras` test is actually happening? Looking at the CI logs for your latest commit, it seems to me that only the `test` one is running.

Reply:
Thanks for checking that! Should be fixed after I push.